form-samples/two-column-logo-header/README.md

```diff
@@ -0,0 +1,39 @@
+# Dashed Line Header
+
+## Summary
+This sample displays the `Title` and `Logo` values in the heading and an additional 'Description' below the 'Title'. If the `Description` value is empty, it will be hidden.
+
+![screenshot of the sample](./assets/screenshot_two_column.png)
+
+
+## Form requirements
+
+|Type                   |Internal Name|Required|
+|-----------------------|-------------|:------:|
+|Single line of text    |Title        |Yes     |
+|Single of text |Description  |No      |
```

Comment:
Please list the columns used in the sample, such as `RequestStatus`, and delete the columns not used.